testing/trino-product-tests/src/main/java/io/trino/tests/product/hive/TestHiveTransactionalTable.java

```diff
@@ -1836,6 +1837,33 @@ public void testDoubleUpdateAndThenReadFromHive()
         });
     }
 
+    @Test(groups = HIVE_TRANSACTIONAL)
+    public void testDeleteWithOriginalFiles()
+    {
+        withTemporaryTable("test_delete_with_no_predicate_on_ctas", true, false, NONE, tableName -> {
+            onTrino().executeQuery("SET SESSION scale_writers = true"); //these 3 properties are necessary to make sure there is more than 1 original file created
+            onTrino().executeQuery("SET SESSION writer_min_size = '4kB'");
+            onTrino().executeQuery("SET SESSION task_writer_count = 2");
+            onTrino().executeQuery(format(
+                    "CREATE TABLE %s WITH (transactional = true) AS SELECT * FROM tpch.sf1000.orders LIMIT 100000", tableName));
```

Comment:
you're using 100K rows here. `tpch.tiny.orders` has 60K rows. Would 60K be sufficient?
use smallest schema possible

Reply:
it looks like it is enough

Reply:
it is not, it worked locally but failed here